README.md

```diff
@@ -3,22 +3,22 @@
 [![Check schema](https://github.com/GCTC-NTGC/gc-digital-talent/actions/workflows/check-lighthouse-schema.yml/badge.svg)](https://github.com/GCTC-NTGC/gc-digital-talent/actions/workflows/check-lighthouse-schema.yml) [![CodeQL](https://github.com/GCTC-NTGC/gc-digital-talent/actions/workflows/codeql-analysis.yml/badge.svg)](https://github.com/GCTC-NTGC/gc-digital-talent/actions/workflows/codeql-analysis.yml) [![Jest Tests](https://github.com/GCTC-NTGC/gc-digital-talent/actions/workflows/jest.yml/badge.svg)](https://github.com/GCTC-NTGC/gc-digital-talent/actions/workflows/jest.yml) [![Lint](https://github.com/GCTC-NTGC/gc-digital-talent/actions/workflows/lint.yml/badge.svg)](https://github.com/GCTC-NTGC/gc-digital-talent/actions/workflows/lint.yml) [![PHP Unit Tests](https://github.com/GCTC-NTGC/gc-digital-talent/actions/workflows/phpunit.yml/badge.svg)](https://github.com/GCTC-NTGC/gc-digital-talent/actions/workflows/phpunit.yml) [![bundlewatch](https://github.com/GCTC-NTGC/gc-digital-talent/actions/workflows/bundlewatch.yml/badge.svg)](https://github.com/GCTC-NTGC/gc-digital-talent/actions/workflows/bundlewatch.yml) [![e2e-tests](https://github.com/GCTC-NTGC/gc-digital-talent/actions/workflows/e2e-tests.yml/badge.svg)](https://github.com/GCTC-NTGC/gc-digital-talent/actions/workflows/e2e-tests.yml) [![Translations](https://github.com/GCTC-NTGC/gc-digital-talent/actions/workflows/translations.yml/badge.svg?branch=main)](https://github.com/GCTC-NTGC/gc-digital-talent/actions/workflows/translations.yml)
 
 The GC Digital Talent app is divided into multiple services, each treated as its own sub-project:
+
 - `/api`, the API service
 - `/apps`, (new) location for frontend applications
-- `/packages`, code shared by multiple other workspaces
+- `/packages`, npm packages used by the `/apps`
```

Comment:
lol, "the /apps" sounds funny. (I know I wrote it). Not a deal breaker but maybe drop "the" or change to:

```suggestion
- `/packages`, npm packages used with `/apps`
```